### ISSUE TITLE
parameter prefixing is incompatible with Neo4j Browser 4.2.5

### ISSUE THREAD

**Issue by terryf82** [neo4j-graphql/neo4j-graphql-js#612]
I understand this library has been deprecated, in favour of an official Neo4j [library](https://github.com/neo4j/graphql/). The original author has explained his intention to continue development [under a fork](https://github.com/neo4j-graphql/neo4j-graphql-js/issues/608), but there appears to be no way to create issues on his repo.

This library has a practice of prefixing query parameters, when they need to be passed down into nested resolvers. For example, this query and parameter set:
```
query (
  $destId: String!
  $posId: Int!
) {
  Destination(vendorId: $destId) {
    name(posId: $posId)
  }
}

{ "destId": "553248634383041473", "posId": 1}
```

results in this generated cypher query:
```
MATCH (`destination`:`Destination` {vendorId:$vendorId}) RETURN `destination` {name: apoc.cypher.runFirstColumn(“MATCH (this)-[r:APPEARS_ON]->(p:PointOfSale)
WHERE p.posId = $posId
RETURN r.name”, {this: destination, posId: $1_posId}, false)} AS `destination`
```

`$posId` is copied into `$1_posId` when it needs to be passed into the nested resolver for `name`. For several years at least this has worked fine, but as of Neo4j Browser 4.2.5 (now the default for Aura clusters) the cypher query parser fails to handle the `$1_posId` parameter, because according to the Neo4j [docs](https://neo4j.com/docs/cypher-manual/current/syntax/parameters/#cypher-parameters) parameters "...cannot start with a number or a currency symbol." This seems to be newly enforced in 4.2.5, in a way that it wasn't in 4.2.2.

Are there any options here? Upgrading to the new Neo4j library is not an option right now, due to the significant changes in schema [output](https://neo4j.com/docs/graphql-manual/current/guides/migration-guide/queries/) that generates. Could this library (or its active fork) perhaps have a hotfix applied to use a different approach to naming nested parameters, e.g. with a `_1` suffix rather than `1_` prefix?

**Comment by johnymontana** [comment]
Hi @terryf82 - I was able to reproduce this using an Aura instance. 

I've just published a new release of this package v2.19.3 that escapes Cypher parameters and have tested that it works with Aura. Could you please test with this version and let us know if that solves your issue?

See ce5d1a71e73f164f97d17c8df07bbcb4d05403a2

**Comment by terryf82** [comment]
@johnymontana many thanks, it looks like your changes have solved the issue.
